.github/workflows/release-crypto-icons.yml

```diff
@@ -0,0 +1,44 @@
+name: "[Release] crypto icons"
```

Comment:
I don't like the name crypto icons lets name it `[Release] suite coin icons`

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/12694/commits/8489b3c8d033ea65d640e0e8d9a361864c24ab55